besu/src/main/java/org/hyperledger/besu/cli/options/unstable/NativeLibraryOptions.java

```diff
@@ -53,7 +53,7 @@ public class NativeLibraryOptions {
           "Per default a native library is used for modexp. "
```

Comment:
If the change in line 56 is correct this description needs to be changed to default `false`